tests/results/1e601c4aa29b6de728ff62b5c7d56b918a309b1416aa22f528b722732f396084/result.json

```diff
@@ -62,8 +62,8 @@
   "files": {
     "extracted": [
       {
-        "name": "bda4205533_deobfuscated.bat",
-        "sha256": "bda420553303de9d7a0cb8b9983b4066bbcd01829fbc6cda89510fee5742e1f8"
+        "name": "b32a84d155_deobfuscated.bat",
```

Comment:
Should the associated file be renamed too? (bda4205533_deobfuscated to b32a84d155_deobfuscated)